### PR TITLE
[caps] Add LNL caps

### PR DESCRIPTION
**Pull request by Hangjie22Coder**

### CODE REVIEW

**Comment by Hangjie22Coder** [comment]
Hi @Bin-CI @FocusLuo @uartie, can you review this for me?

File: lib/caps/LNL/info [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+info = dict(
+  gpu = dict(gen = 13),
```
Comment (FocusLuo) [inline lib/caps/LNL/info:12]: LNL is Gen 15

Comment (Hangjie22Coder) [reply lib/caps/LNL/info]: Thank you for the review, it has been updated.

**Review by FocusLuo — APPROVED** [review]
LGTM

**Pull request merged**